repository.py

```diff
@@ -939,7 +940,7 @@ def listPackagesFromRepos(repos, rpm_pattern, query_format='%{nevr}'):
         repo._accessor.start()
 
     try:
-        createMainYumConfig(yum_conf_path, repos, cachedir)
+        createMainYumConfig(yum_conf_path, repos, cachedir, override_repogpgcheck=0)
```

Comment:
Until I reach this line I was convinced that it was a bool, based on the name that sounds like an action.

Reply:
`repogpgcheck_override` would be better?

Reply:
as a matter of fact this "overriding a yum-bool, or `None` for not overriding"

Reply:
applied